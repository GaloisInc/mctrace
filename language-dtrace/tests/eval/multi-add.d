int x;
:::read {
  x = x + 1;
}
