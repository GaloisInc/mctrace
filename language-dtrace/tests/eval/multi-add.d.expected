EvalExpected { postState = [("x", RegBV<32> 3)]
             , probeSeq = [base:::read, base:::read, base:::read]
             }
